config/ingest.go

```diff
@@ -40,6 +41,12 @@ func (c KafkaIngestConfiguration) CreateKafkaConfig() kafka.ConfigMap {
 		"go.logs.channel.enable": true,
 	}
 
+	// This is needed when using localhost brokers on OSX,
+	// since the OSX resolver will return the IPv6 addresses first.
+	if strings.Contains(c.Broker, "localhost") || strings.Contains(c.Broker, "127.0.0.1") {
+		config["broker.address.family"] = "v4"
+	}
```

Comment:
This is okay temporarily, but I can imagine a scenario where users want ipv6 resolution for localhost.

I'd suggest making this setting available in config for the long-term.

Removing localhost and using the loopback IP address would also be okay.

Reply:
Opened an issue and added to comment: https://github.com/openmeterio/openmeter/issues/321